org.eclipse.buildship.oomph.test/gradle.properties

```diff
@@ -0,0 +1,2 @@
+# set SWTBot to put screenshots from failed tests under the build folder
+systemProp.org.eclipse.swtbot.screenshots.dir=build/screenshots
```

Comment:
Does it works if we move it to the root project's gradle.properties file?

Reply:
ah, this is a relic from copying the framework from `org.eclipse.buildship.ui.test`. I'll remove the whole file.